rholang/src/main/scala/coop/rchain/rholang/interpreter/RholangCLI.scala

```diff
@@ -179,14 +182,20 @@ object RholangCLI {
     println(s"Compiled $fileName to $binaryFileName")
   }
 
-  def evaluatePar(runtime: Runtime[Task])(par: Par)(implicit scheduler: Scheduler): Unit = {
+  def evaluatePar(runtime: Runtime[Task], quiet: Boolean = false)(
```

Comment:
please make this private

Reply:
the 'add quiet flag' change should be a separate commit.

Reply:
perhaps all the changes to rholang-cli should be a separate PR

Reply:
separate PR or this PR but in separate commits - I'm fine either way :)